src/dbnode/ts/writes/write_batch_pool.go

```diff
@@ -25,13 +25,10 @@ import (
 )
 
 const (
-	// defaultInitialiBatchSize determines the initial batch size that will be used when filling up the
-	// pool.
-	defaultInitialBatchSize = 1024
 	// defaultWritePoolMaxBatchSize is the default maximum size for a writeBatch that the pool
 	// will allow to remain in the pool. Any batches larger than that will be discarded to prevent
 	// excessive memory use forever in the case of an exceptionally large batch write.
-	defaultMaxBatchSize = 100000
+	defaultMaxBatchSize = 10240
```

Comment:
Reducing this default as 100000 seems to be crazy lot (it would result in a 39MB batch).